### PR TITLE
Feature 67 edit resource page

### PR DESCRIPTION
**Pull request by BeckyRushton**
Created edit resource and request resource pages, as well as creating form component. Tests completed for all and styling for responsiveness.

### CODE REVIEW

File: src/components/Form/Form.scss [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    &__auto-purchase {
+      grid-area: auto-purchase;
+      padding: 10px;
+    }
+    &__quantity-remaining {
+      grid-area: quantity-remaining;
+      padding: 10px;
+    }
+  }
+  .button {
```
Comment (TristanPere) [inline src/components/Form/Form.scss:144]: Never call _.button_ as a styling convention as it restyles every appearance of a button. put your buttons in a named html container and style that.

File: src/components/Form/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import Button from "../Button/Button";
+import DropdownField from "../DropdownField/DropdownField";
+import InputField from "../InputField/InputField";
+import "./Form.scss";
+
+const Form = ({
```
Comment (TristanPere) [inline src/components/Form/Form.jsx:7]: Please call this something other than form as multiple forms are used and this isn't specific enough to this case. Same goes for file name.

File: src/components/Form/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    </div>
+  );
+
+  const optionsJSX = optionsArr.map((option, index) => {
+    return (
+      <option className="option" value={option} key={index}>
+        {option}
+      </option>
+    );
+  });
```
Comment (TristanPere) [inline src/components/Form/Form.jsx:68]: Is this necessary to have a map for two options? cant we just right the options array?

File: src/components/Form/Form.jsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+          </div>
+        </div>
+      </div>
+      <div className="form-container__auto-purchase">
+        <InputField label="Auto-Purchase Level" placeholderText="50" />
+      </div>
+      <div className="form-container__quantity-remaining">
+        <InputField label="Quantity Remaining" placeholderText="500" />
+      </div>
+      <Button buttonText={text} />
```
Comment (TristanPere) [inline src/components/Form/Form.jsx:113]: please put a div around this for styling

File: src/components/InputField/InputField.jsx [hunk 5]
```diff
@@ -2,13 +2,13 @@ import React from 'react'
 import "./InputField.scss"
 
 const InputField = (props) => {
-    const {label, placeholderText, id, handleInput} = props
+    const {label, placeholderText, id, handleInput, readOnly} = props
```
Comment (TristanPere) [inline src/components/InputField/InputField.jsx:5]: inline props please

**Review by TristanPere — CHANGES_REQUESTED** [review]
Great form functionality, matches the figma perfectly with the addition of the extra input we talked about. Please make changes noted as well as address some weird styling issues at tablet display width

**Review by hannahmitchell99 — CHANGES_REQUESTED** [review]
all looking great, just add in the separate array for the request resource page as discussed

**Review by hannahmitchell99 — APPROVED** [review]

**Review by TristanPere — APPROVED** [review]

**Pull request merged**